src/stdlib/String.ts

```diff
@@ -1,36 +1,35 @@
-import { BrsType, Callable, ValueKind, BrsString, Int32 } from "../brsTypes";
-import { Interpreter } from "../interpreter";
+import { BrsType, Callable, ValueKind, BrsString, Int32 } from '../brsTypes';
+import { Interpreter } from '../interpreter';
 
 /** Converts the string to all uppercase. */
 export const UCase = new Callable(
     {
-        name: "UCase",
-        args: [{ name: "s", type: ValueKind.String }],
-        returns: ValueKind.String,
+      name: 'UCase',
+      args: [{name: 's', type: ValueKind.String}],
+      returns: ValueKind.String
     },
     (interpreter: Interpreter, s: BrsString) => new BrsString(s.value.toUpperCase())
 );
 
 /** Converts the string to all lowercase. */
 export const LCase = new Callable(
     {
-        name: "LCase",
-        args: [{ name: "s", type: ValueKind.String }],
-        returns: ValueKind.String,
+      name: 'LCase',
```

Comment:
Looks like the IDEA products default to two-space indents and single quotes, but the convention in these files has typically been four-space indents and double quotes.  That's *entirely* on me, because I never set up a `.editorconfig` or anything like it!

Would it be too much trouble to reformat after I set one up tonight?  I'd normally merge this as-is and reformat later, but I'd much rather you show up on these lines in `git blame` than me! :smile:

Reply:
Sorry about that, it is defaulted at 2. I do not mind converting back. (I will also change the default for working on this project)